python/lsst/ts/ofc/ofc_data/ofc_data.py

```diff
@@ -100,8 +100,14 @@ class OFCData(BaseOFCData):
         If input `config_dir` does not exists.
     """
 
-    def __init__(self, name=None, config_dir=None, log=None, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(
+        self,
+        name: str | None = None,
+        config_dir: str | None = None,
+        log: logging.Logger | None = None,
+        **kwargs: object,
```

Comment:
For the `**kwargs` or `*args`, please consider the followings and do the related tuning:

https://github.com/lsst-ts/ts_m2gui/blob/develop/python/lsst/ts/m2gui/utils.py#L426-L431

This comment applies to others.